test/metabase/api/search_test.clj

```diff
@@ -503,6 +503,16 @@
                (binding [*search-request-results-database-id* db-id]
                  (search-request-data user :q "Round Table"))))))))
 
+(deftest all-users-no-data-perms-table-test
+  (testing (str "If the All Users group doesn't have perms to view a Table they sholdn't see it (#16855)")
```

Comment:
don't think you need the `str` here

Reply:
Good call, I totally copied it from the above (which has multiple strings concatenated together) and then edited away the other strings.